libr/egg/egg_lang.c

```diff
@@ -3,67 +3,102 @@
 #include <r_egg.h>
 
 #define CTX egg->context
-char *nested[32] = {0};
-char *nestede[32] = {0};
+char *nested[32] = {NULL};
```

Comment:
please indent the file with r2-indent or using tabs with default indentation of vim, , the whole tab->space diff makes it impossible to review